Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.17.3
```

Comment:
Better to use the same golang version used in go.mod

Reply:
I think i used this one because it's mentioned in the libvirt instructions maybe we should bump?

Reply:
We are bumping the version to 1.18. So you can change this to `1.18`